### ISSUE TITLE
upgrade the legend indicator

### ISSUE THREAD

**Issue by wahdina** [koetjeengdjalanan/NGAGE#1]
Need improve the legend from 3 indicators to 5 indicators, for make its more detail. 
and apply it to all the worksheet. 

here the detail below:
 0 
 50
 70
 70-80 
 >80

**Comment by koetjeengdjalanan** [comment]
Let me rephrase this: 

> There should be an upgrade from 3 to 5 Indicators.

The details for indicator is as such:

```
|--------|---|-|----|
A        B   C D    E

A : 0
B : .5
C : .7
D : .8
E : 1
```

Ruleset to be, Where `χ` is the value of cell:

* `χ` = A : ?
* `χ` <= B : ?
* B < `χ` <= C : ?
* C < `χ` <= D : ?
* D < `χ` <= E : ?

@wahdina Please provide me with the `hex` colour code.

**Comment by wahdina** [comment]
can we try with this code, please 
χ = A :  #006400
χ <= B : #299438
B < χ <= C : #7ECC49
C < χ <= D :  #FF9933
D < χ <= E : #DB4035
